odata-url-conventions/5 Query Options.md

```diff
@@ -90,7 +90,7 @@ http://host/service/People?$filter=Age gt '50'
 ```
 the expression would always evaluate to `null` because the age 50 is erroneously
 given as a string and the [`$filter`](#SystemQueryOptionfilter) would return an empty result, although this is
-really the result of a typing error. That's why a "type mismatch" error SHOULD
+really the result of a typing error. That's why a "type mismatch" error must
```

Comment:
```suggestion
really the result of a typing error. That's why a "type mismatch" error MUST
```
If we really mean it, we MUST use uppercase according to https://www.rfc-editor.org/rfc/rfc8174.html#section-1

Reply:
But not in an example (which is non-normative).